src/app/utils/utils.js

```diff
@@ -66,6 +66,20 @@ function Utils() {
     return selectedTags;
   };
 
+  /**
+   * getFilters(filters)
```

Comment:
You're not really getting filters, you're mapping the filter array to id/label pairs, so maybe a better name?